File: go/fix1A.go
package main

import (
	"fmt"
	"io/ioutil"
	"os"
)

func main() {
	if len(os.Args) > 1 {
		data, err := ioutil.ReadFile(os.Args[1])
		if err != nil {
			fmt.Println(err)
		}
		if data[len(data)-1] == 26 {
			data = data[:len(data)-3]
			werr := ioutil.WriteFile(os.Args[1], data, 0644)
			if werr != nil {
				fmt.Println(werr)
			}
			println("ok")
		}
	}
}

File: node/fix1A.js
var fs = require('fs');
if (process.argv.length == 3) {
    filename = process.argv[2];
    fs.readFile(filename, check4save);
}
function check4save(err, data) {
    if (data[data.length - 1] == 26) {
        fs.writeFile(filename, data.slice(0, data.length - 3), function (err) {
            if (err) throw err;
            console.log('修正完成');
        });
    }
}